### PR TITLE
Reduce errors in checker tests

### PR DESCRIPTION
**Pull request by MorrisJobke**
Before there were errors like this in the logs:

```
Testing testing
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing files_trashbin
Nextcloud is not installed - only a limited number of commands are available
PHP Warning:  fileperms(): stat failed for /drone/src/github.com/nextcloud/server/data/nextcloud.log in /drone/src/github.com/nextcloud/server/lib/private/Log/File.php on line {"reqId":"m58FGv7Y8nH2dWwx6uws","level":3,"time":"2019-03-05T09:30:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"DOMDocument::schemaValidate(): Element 'trash': This element is not expected. Expected is ( public ). at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/App\/CodeChecker\/InfoChecker.php#63","userAgent":"--","version":""}
App is compliant - awesome job!
PHP Warning:  fileperms(): stat failed for /drone/src/github.com/nextcloud/server/data/nextcloud.log in /drone/src/github.com/nextcloud/server/lib/private/Log/File.php on line {"reqId":"m58FGv7Y8nH2dWwx6uws","level":3,"time":"2019-03-05T09:30:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"fileperms(): stat failed for \/drone\/src\/github.com\/nextcloud\/server\/data\/nextcloud.log at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/Log\/File.php#140","userAgent":"--","version":""}
{"reqId":"m58FGv7Y8nH2dWwx6uws","level":3,"time":"2019-03-05T09:30:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"chmod(): No such file or directory at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/Log\/File.php#141","userAgent":"--","version":""}
Testing comments
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing files_versions
Nextcloud is not installed - only a limited number of commands are available
PHP Warning:  fileperms(): stat failed for /drone/src/github.com/nextcloud/server/data/nextcloud.log in /drone/src/github.com/nextcloud/server/lib/private/Log/File.php on line {"reqId":"vSlvqy7mG9CX5m2ZiKcr","level":3,"time":"2019-03-05T09:30:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"DOMDocument::schemaValidate(): Element 'versions': This element is not expected. Expected is ( public ). at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/App\/CodeChecker\/InfoChecker.php#63","userAgent":"--","version":""}
App is compliant - awesome job!
PHP Warning:  fileperms(): stat failed for /drone/src/github.com/nextcloud/server/data/nextcloud.log in /drone/src/github.com/nextcloud/server/lib/private/Log/File.php on line {"reqId":"vSlvqy7mG9CX5m2ZiKcr","level":3,"time":"2019-03-05T09:30:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"fileperms(): stat failed for \/drone\/src\/github.com\/nextcloud\/server\/data\/nextcloud.log at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/Log\/File.php#140","userAgent":"--","version":""}
{"reqId":"vSlvqy7mG9CX5m2ZiKcr","level":3,"time":"2019-03-05T09:30:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"chmod(): No such file or directory at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/Log\/File.php#141","userAgent":"--","version":""}
Testing files_sharing
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing encryption
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing systemtags
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing user_ldap
Nextcloud is not installed - only a limited number of commands are available
Deprecated file found: /drone/src/github.com/nextcloud/server/apps/user_ldap/appinfo/update.php - please use repair steps
App is compliant - awesome job!
Testing updatenotification
Nextcloud is not installed - only a limited number of commands are available
PHP Warning:  fileperms(): stat failed for /drone/src/github.com/nextcloud/server/data/nextcloud.log in /drone/src/github.com/nextcloud/server/lib/private/Log/File.php on line {"reqId":"9DixBfRGZk3TxDBSewBN","level":3,"time":"2019-03-05T09:30:14+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"DOMDocument::schemaValidate(): Element 'commands': This element is not expected. Expected is one of ( activity, dashboard, fulltextsearch, navigations, contactsmenu, collaboration, sabre, public ). at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/App\/CodeChecker\/InfoChecker.php#63","userAgent":"--","version":""}
App is compliant - awesome job!
PHP Warning:  fileperms(): stat failed for /drone/src/github.com/nextcloud/server/data/nextcloud.log in /drone/src/github.com/nextcloud/server/lib/private/Log/File.php on line {"reqId":"9DixBfRGZk3TxDBSewBN","level":3,"time":"2019-03-05T09:30:14+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"fileperms(): stat failed for \/drone\/src\/github.com\/nextcloud\/server\/data\/nextcloud.log at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/Log\/File.php#140","userAgent":"--","version":""}
{"reqId":"9DixBfRGZk3TxDBSewBN","level":3,"time":"2019-03-05T09:30:14+00:00","remoteAddr":"","user":"--","app":"PHP","method":"","url":"--","message":"chmod(): No such file or directory at \/drone\/src\/github.com\/nextcloud\/server\/lib\/private\/Log\/File.php#141","userAgent":"--","version":""}
Testing twofactor_backupcodes
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing oauthNextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing federatedfilesharing
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
```

After that it should more look like this:

```
Testing testing
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing files_trashbin
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing comments
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing files_versions
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing files_sharing
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing encryption
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing systemtags
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing user_ldap
Nextcloud is not installed - only a limited number of commands are available
Deprecated file found: /drone/src/github.com/nextcloud/server/apps/user_ldap/appinfo/update.php - please use repair steps
App is compliant - awesome job!
Testing updatenotification
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing twofactor_backupcodes
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing oauthNextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
Testing federatedfilesharing
Nextcloud is not installed - only a limited number of commands are available
App is compliant - awesome job!
```

### CODE REVIEW

File: autotest-checkers.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ RESULT=$(($RESULT+$?))
 php ./build/htaccess-checker.php
 RESULT=$(($RESULT+$?))
 
-
+mkdir data/
```
Comment (nickvergessen) [inline autotest-checkers.sh]: check if it does exist? and only delete if not?
To not break dev enviroments?

Comment (MorrisJobke) [reply autotest-checkers.sh]: Makes sense 👍

Comment (MorrisJobke) [reply autotest-checkers.sh]: Done

**Review by nickvergessen — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]
Oh yes please!

**Review by juliusknorr — APPROVED** [review]

**Pull request merged**